GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
+        if let colorValue = keyColorPair.1?.rgbColor?.hexValue {
+          addColorWithName(colorName, value: colorValue)
+        }
+      }
+    }
+  }
+  
+  public func addColorWithName(name: String, value: String) {
+    addColorWithName(name, value: ColorsTextFileParser.parse(value))
```

Comment:
Not sure this method belongs to `CLRFileParser`, probably a copy/pasted bit of code you forgot to remove?